CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Django Field Audit change log
+
+## v0.2 - 2022-05-24
+
+- Initial implementation.
+- There is no version `0.1`.
```

Comment:
Because I burned version `0.1` by accidentally uploading it to my personal PyPi account (and you can't push the same version of the same package more than once, even if the original package has been deleted. :facepalm: In order to upload the package to the Dimagi account, I had to bump the version (in hindsight I assume `0.1.1` would have been sufficient, but I happened to try `0.2` because I didn't actually expect it to work).